rpcclient/mining.go

```diff
@@ -100,3 +105,55 @@ func (r FutureGetBlockTemplateResult) Receive() (*rpcmodel.GetBlockTemplateResul
 func (c *Client) GetBlockTemplate(payAddress string, longPollID string) (*rpcmodel.GetBlockTemplateResult, error) {
 	return c.GetBlockTemplateAsync(payAddress, longPollID).Receive()
 }
+
+// ParseBlock Accepts a GetBlockTemplateResult and parse it into a Block
+func ParseBlock(template *rpcmodel.GetBlockTemplateResult) (*util.Block, error) {
```

Comment:
I prefer a name that implies it converts a GetBlockTemplateResult into a block

Reply:
`ParseGetBlockTemplateResult`?

Reply:
Done `ConvertGetBlockTemplateResultToBlock`